app/controllers/listener/lines_controller.rb

```diff
@@ -26,14 +26,10 @@ def react(event)
     intro_events = %w(join)
     case @event_type
     when 'message'
-      if group_chat?
-        check_rumor
-      else # 1-on-1 chat
-        if about_coronavirus?
-          answer_query
-        else
-          check_rumor
-        end
+      if !group_chat? and about_coronavirus?
```

Comment:
```suggestion
      if !group_chat? && about_coronavirus?
```
ruby use `&&` for and, and `and` is more like flow controlling (?)
https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby